independent-publisher-connectors/Working days/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2104 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.2.0",
+    "title": "Working days",
+    "description": "Working days API. It covers more than 45 countries and 230 regional calendars. It also covers main stock exchanges calendars. The quality of data is the top priority. API offers up to date information by following government announcements regarding public holidays changes.",
+    "contact": {
+      "name": "Tomasz Poszytek",
+      "url": "https://aka.ms/poszytek",
+      "email": "tomasz@poszytek.eu"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://api.workingdays.org/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://api.workingdays.org/1.2/privacy_policy.php"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Finance;Commerce"
+    }
+  ],
+  "host": "api.workingdays.org",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/1.2/add_working_days": {
+      "get": {
+        "summary": "Add working days",
+        "description": "Add (or remove) any number of working days to a date.",
+        "operationId": "Add_working_days",
```

Comment:
@tposzytek please use pascal case for operationId properties. More information can be found in [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).

Reply:
Checked everywhere and fixed.